back/engines/commercial/admin_api/app/controllers/admin_api/tenants_controller.rb

```diff
@@ -27,14 +27,15 @@ def create
     end
 
     def update
-      success, tenant, config = tenant_service.update_tenant(@tenant, legacy_tenant_params)
-      if success
-        tenant_json = AdminApi::TenantSerializer.new(tenant, app_configuration: config).to_json
-        render json: tenant_json, status: :ok
-      else
-        tenant.errors.merge!(config.errors)
-        render json: { errors: tenant.errors.details }, status: :unprocessable_entity
-      end
+      update_tenant legacy_tenant_params
+    end
+
+    def remove_locale
```

Comment:
But I agree it's controversial, so up to you.

Reply:
For me it's really about removing the locale, not about replacing it. Unfortunately removing a locale that is still used by users is not possible today, that's why the data job was introduced (to avoid support work). So replacing is more a side effect for removing the locale and it's necessary to provide the locale that will replace the removed locale in those cases.

Reply:
Maybe you're right, sorry for the noise.
But maybe let's at least rename `remove_locale` parameter to `removing_locale`:
1. To distinguish it from `remove_locale` method/action
2. To make it more similar to `replacing_locale` parameter

But again, it's fine as it is now too.